Emby.Naming/AudioBook/AudioBookNameParserResult.cs

```diff
@@ -14,5 +16,50 @@ public struct AudioBookNameParserResult
         /// Gets or sets optional year of release.
         /// </summary>
         public int? Year { get; set; }
+
+        /// <summary>
+        /// Comparison equallity operators for <see cref="AudioBookNameParserResult"/>.
+        /// </summary>
+        /// <param name="left">Source comparison object.</param>
+        /// <param name="right">Target comparision object.</param>
+        /// <returns>If both objects are the equal.</returns>
+        public static bool operator ==(AudioBookNameParserResult left, AudioBookNameParserResult right)
+        {
+            return left.Equals(right);
+        }
+
+        /// <summary>
+        /// Comparison unequallity operators for <see cref="AudioBookNameParserResult"/>.
```

Comment:
```suggestion
        /// Inequality operator for <see cref="AudioBookNameParserResult"/>.
```

Same of the other occurrences